packages/jest-cli/package.json

```diff
@@ -4,8 +4,10 @@
   "version": "22.4.2",
   "main": "build/jest.js",
   "dependencies": {
+    "@babel/highlight": "^7.0.0-beta.46",
     "ansi-escapes": "^3.0.0",
     "chalk": "^2.0.1",
+    "easy-table": "^1.1.1",
```

Comment:
seems small enough, happy to take suggestions about others

Reply:
or other ideas on how to print the info

Reply:
Stack trace with code frame maybe, but this is OK for now

Reply:
Ooooh, faking a stack trace with this info, I like the idea! I'll give it a whack later today